books_manager.rb

```diff
@@ -0,0 +1,58 @@
+require 'json'
+require './book'
+
+class BooksManager
+  attr_reader :books
+
+  def initialize
+    books = File.read('./data/books.json')
+    @books = books == '' ? [] : convert_hashs_to_books(JSON.parse(books))
+  end
+
+  def add_book
+    puts 'Enter publisher'
+    publisher = gets.chomp
+    puts 'Enter cover state'
+    cover_state = gets.chomp
+    puts 'Enter publish date'
+    publish_date = gets.chomp
+    puts 'Enter true or false if the books is archived '
+    archived = gets.chomp
+    book = Book.new(publisher, cover_state, publish_date, archived: archived)
```

Comment:
Consider creating the object without the `archived` variable, then use the `move_to_archive` method in the class.
Also, you should convert Y/N from user input to true/false for the object